mainwindow.cpp

```diff
@@ -17,58 +19,44 @@ MainWindow::~MainWindow()
 
 void MainWindow::on_rockButton_clicked()
 {
-    player1->choice = ROCK;
-    ui->paperButton->setEnabled(false);
-    ui->scissorsButton->setEnabled(false);
+    player1->choice = 0;
     display_round();
+    player1->done_chosing(player1, ui->player1Label);
 }
 
 
 void MainWindow::on_paperButton_clicked()
 {
-    player1->choice = PAPER;
-    ui->rockButton->setEnabled(false);
-    ui->scissorsButton->setEnabled(false);
+    player1->choice = 1;
```

Comment:
I think we can keep the choice enum and use it in the map instead of using number

Reply:
got it

Reply:
done.